router/cohorts-router.js

```diff
@@ -0,0 +1,122 @@
+const router = require('express').Router();
+const knex = require('knex');
+
+const knexConfig = require('../knexfile');
+
+const db = knex(knexConfig.development);
+
+// POST REQUEST
+router.post('/cohorts', (req, res) => {
+    db('cohorts')
+        .insert(req.body)
+        .then(cohortId => {
+            const [id] = cohortId;
+
+            db('cohorts')
+                .where({ id })
+                .then(cohort => {
+                    res.status(201).json(cohort)
+                })
+                .catch(err => {
+                    res.status(500).json(err)
+                })
+        })
+        .catch(err => {
+            res.status(500).json(err)
+        })
+})
+
+// GET REQUEST
+router.get('/cohorts', (req, res) => {
+    db('cohorts')
+        .then(cohorts => {
+            res.status(200).json(cohorts)
+        })
+        .catch(err => {
+            res.status(500).json(err)
+        })
+})
+
+router.get('/cohorts/:id', (req, res) => {
+    const id = req.params.id;
+
+    db('cohorts')
+        .where({ id: id })
+        .then(cohort => {
```

Comment:
Check to make sure cohort is not empty and that that id matched a row in the data.